data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/ConnectionConfigurationTests.java

```diff
@@ -397,16 +323,10 @@ void testCreateClientWithAWSSigV4AndSTSRole() {
     }
 
     @Test
-    void testCreateOpenSearchClientWithAWSSigV4AndSTSRole() {
-        final PluginSetting pluginSetting = generatePluginSetting(
-                TEST_HOSTS, null, null, null, null, true, null, TEST_ROLE, TEST_CERT_PATH, false);
+    void testCreateOpenSearchClientWithAWSSigV4AndSTSRole() throws IOException {
+        final OpenSearchSinkConfig openSearchSinkConfig = generateOpenSearchSinkConfig(AWS_WITH_CERT_AND_ARN);
         final ConnectionConfiguration connectionConfiguration =
-                ConnectionConfiguration.readConnectionConfiguration(pluginSetting);
```

Comment:
Ideally, we should keep the tests same as much as possible to make sure that the changes have not caused any regression. Please just change pluginSetting with OpenSearchSinkConfig and keep the rest of the test same as before.